packages/ui-logic/src/random.ts

```diff
@@ -2,4 +2,6 @@ import PseudoRandom, { MersenneTwister19937, string } from "random-js";
 
 const prng: PseudoRandom.Engine = MersenneTwister19937.autoSeed();
 const pool = "abcdefghijklmnopqrstuvwxyz0123456789";
-export const randomString = (len: number): string => string(pool)(prng, len);
+export function randomString(len: number): string {
+  return string(pool)(prng, len);
+}
```

Comment:
👍 

Can you create a simple unittest for everything that is in `ui-logic`? It does not need to covery every edge case but at least run `randomString` to make sure the dependencies are in place